ops/cli.py

```diff
@@ -160,6 +160,21 @@ def run_cloudsql_proxy(project: str, cloudsql_instance_name):
                         f'-token="{token}" -instances="{instance}=tcp:0.0.0.0:5432"'])
     return subprocess.check_output(docker_command, shell=True, encoding='utf-8').strip()
 
+def infer_cloudsql_name(project: str):
+    """Try to find a single cloudsql instance with a truthy 'wfl' tag."""
+    info("=> Finding cloud_sql_proxy instance")
+    gcloud_command = f"gcloud --format=json --project {project} sql instances list"
```

Comment:
You can add `--filter="settings.userLabels.wfl=false"` to filter by label so that you don't have to do that with python: https://cloud.google.com/sdk/gcloud/reference/topic/filters

Reply:
Welp. Thank you.